draft-ietf-scitt-architecture.md

```diff
@@ -126,63 +126,47 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 
 # Introduction
 
-This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
-It achieves this goal by enforcing the following complementary security guarantees:
-
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+This document describes the scalable, flexible, and decentralized SCITT architecture.
+Its goal is to enhance auditability and accountability across supply chains.
+
+In supply chains, there are producers and consumers of items.
+Consumers may like to have information about the items that they consume, such as an item's provenance.
+SCITT provides a way for consumers to obtain this information in a way that is "transparent", that is, producers cannot lie about the information without it being detected.
+SCITT achieves this by having producers (also called Issuers) publish information in a Transparency Service, where consumers (also called Verifiers) can check the information.
+
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts they release, for example: "Issuer name", "Artifact type", "Artifact version".
+1. Issuers sign the Statements, producing Signed Statements.
+1. Issuers submit the Signed Statements to Transparency Services.
+1. Transparency Services take the submitted Signed Statements and insert them into an Append-only Log, producing Transparent Statements.
```

Comment:
I'm assuming lines 144-147 are additional text to the numbered bullet on line 143, rather than additional numbered lines? Could the numbered items be summarized more succinctly?